plugins/modules/system/java_cert.py

```diff
@@ -369,30 +467,61 @@ def main():
     if not keystore_create:
         test_keystore(module, keystore_path)
 
-    cert_present = check_cert_present(module, executable, keystore_path,
-                                      keystore_pass, cert_alias, keystore_type)
+    alias_exists, alias_exists_output = _check_cert_present(
+        module, executable, keystore_path, keystore_pass, cert_alias, keystore_type)
```

Comment:
What happens if the alias has not been specified? It looks to me that your code is assuming (more than the old code) that alias is always specified.

Reply:
fixed, I don't see the difference with the old code (the one coming from the PR I copied and that was not reviewed before to be put in the trash when migrating to community collections)
I added a check to have at least cert_url or cert_alias to be declared 
```
        required_if=[['state', 'present', ('cert_path', 'cert_url', 'pkcs12_path'), True],
                     ['state', 'absent', ('cert_url', 'cert_alias'), True]],
```

Reply:
If you can please inform if it's enough and  resolve the conversation accordingly

Reply:
Looks ok to me.